docs/structure.md

```diff
@@ -28,6 +28,9 @@ More in-depth structure:
 .
 ├── compose.yaml
 ├── CONTRIBUTING.md
+├── dist
+│   ├── cli_surf-0.1.0-py3-none-any.whl
+│   └── cli_surf-0.1.0.tar.gz
```

Comment:
Did we need the information in this dist directory?

Reply:
@K-dash whoops, we do not. Good catch, I just got rid of that

Reply:
LGTM🚀